docs/production-deployment/self-hosted-guide/deployment.mdx

```diff
@@ -84,18 +76,22 @@ docker run
     temporalio/server:<tag>
 ```
 
-For more details, see the [Docker source file](https://github.com/temporalio/temporal/tree/master/docker).
+The environment variables supported by the Temporal Docker images are documented [on Docker Hub](https://hub.docker.com/r/temporalio/auto-setup).
 
-Each Temporal Server release also ships a `Server with Auto Setup` Docker image that includes an [auto-setup.sh](https://github.com/temporalio/docker-builds/blob/main/docker/auto-setup.sh) script.
+Each Temporal Server release ships an [Auto Setup](https://temporal.io/blog/auto-setup) Docker image that includes an [auto-setup.sh](https://github.com/temporalio/docker-builds/blob/main/docker/auto-setup.sh) script.
 We recommend using this script for initial schema setup of each supported database.
-You should familiarize yourself with [what auto-setup does](https://temporal.io/blog/auto-setup), because you will likely replace every part of the script to customize it for your own infrastructure and tooling choices.
```

Comment:
Having this blog post link is nice for another way for people to get the information unless you don't think it's relevant or accurate anymore.

Reply:
Yup, I just moved it up :)